tests/unit/moduleapi/mallocsize.tcl

```diff
@@ -4,8 +4,18 @@ set testmodule [file normalize tests/modules/mallocsize.so]
 start_server {tags {"modules"}} {
     r module load $testmodule
 
-    test {MallocSize of raw, string and dict} {
-        assert_equal [r mallocsize.set key 12 abc k1 v1 k2 v2] {OK}
-        assert_morethan [memory_usage key] 100
+    test {MallocSize of raw bytes} {
+        assert_equal [r mallocsize.setraw key 40] {OK}
+        assert_morethan [memory_usage key] 0
```

Comment:
If we know we allocated 40 bytes, let's assert that the memory usage is at least 40 bytes?